From 08236738fca59f511f980992a82ec495e8c370d8 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
Date: Wed, 5 Nov 2008 11:03:34 -0600
Subject: [PATCH] ENGR00097778-1 MX51: machine type mismatches with registed name in ARM web

Name used in arch/arm/mach-mx51/Kconfig is MACH_MX51_3STACK.
Name in arm web is MACH_MX51_3DS It make machine_is_mx51_3ds() macro define 0
in 3stack board.

Signed-off-by: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
---
 arch/arm/configs/imx51_3stack_defconfig |    2 +-
 arch/arm/mach-mx51/Kconfig              |    2 +-
 arch/arm/mach-mx51/Makefile             |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx51_3stack_defconfig b/arch/arm/configs/imx51_3stack_defconfig
index e4fe614..e68667d 100644
--- a/arch/arm/configs/imx51_3stack_defconfig
+++ b/arch/arm/configs/imx51_3stack_defconfig
@@ -187,7 +187,7 @@ CONFIG_ARCH_MXC_HAS_NFC_V3=y
 # MX51 Options
 #
 CONFIG_MX51_OPTIONS=y
-CONFIG_MACH_MX51_3STACK=y
+CONFIG_MACH_MX51_3DS=y
 CONFIG_ARCH_MXC_HAS_NFC_V3_2=y
 
 #
diff --git a/arch/arm/mach-mx51/Kconfig b/arch/arm/mach-mx51/Kconfig
index e7e7b69..7655de1 100644
--- a/arch/arm/mach-mx51/Kconfig
+++ b/arch/arm/mach-mx51/Kconfig
@@ -8,7 +8,7 @@ config MX51_OPTIONS
 	select USB_ARCH_HAS_EHCI
 	select MXC_TZIC
 
-config MACH_MX51_3STACK
+config MACH_MX51_3DS
 	bool "Support MX51 3-Stack platforms"
 	default y
 	help
diff --git a/arch/arm/mach-mx51/Makefile b/arch/arm/mach-mx51/Makefile
index 9aa96e7..cdd2fa5 100644
--- a/arch/arm/mach-mx51/Makefile
+++ b/arch/arm/mach-mx51/Makefile
@@ -9,7 +9,7 @@ obj-y   := system.o iomux.o cpu.o mm.o clock.o devices.o serial.o dma.o lpmodes.
 
 obj-$(CONFIG_CPU_V7)		+= wfi.o suspend.o
 
-obj-$(CONFIG_MACH_MX51_3STACK) += mx51_3stack.o mx51_3stack_gpio.o
+obj-$(CONFIG_MACH_MX51_3DS) += mx51_3stack.o mx51_3stack_gpio.o
 
 obj-$(CONFIG_USB_EHCI_ARC_H1)   += usb_h1.o
 
-- 
1.5.4.4

